acceptance_process.md

```diff
@@ -81,25 +54,16 @@ The objective of the Network Steward is to make funding available for projects w
+* Did the applicant declare and control for any potential conflicts of interest ?
 
-## Counter-proposal and Amendment Criteria
-Counter-proposals and amendments to existing proposals are compared using the 5 evaluation criteria but in addition, they must achieve the status of significantly similar and a significant improvement.
-
-### Significantly similar
-A counter-proposal or amendment should be considered to be significantly similar in the event that it targets the same objectives as the original proposal. Some questions which may be useful to the evaluation of significantly similar status include:
-* Do the two proposals have the same short and long term impacts ?
-* Assuming both projects were funded, would it be nonsensical for a single entity to use the results of both projects ?
-
-### A significant improvement
-The definition of a significant improvement is designed to prevent "stealing projects" by submission of proposals which are trivially less costly. While the exact definition of a significant improvement will be left to the discretion of the NS team, applicants can generally consider that:
-* A proposal of otherwise equal value but at more than 5% less cost is a significant improvement
-* A proposal which is not only better than the original, but in fact also better than another project which is better than the original is a significant improvement
-Counter-proposers should be aware that a copy/paste counter-proposal will generally not be considered to be equal to the original without a justification of domain knowledge equal to that which is justified by the original applicant through the text of the proposal.
-
-### Unacceptable projects
+## Unacceptable projects
 Some projects may be judged by the Network Steward to be unacceptable under any conditions. An intuitive example would be a proposal to build an assasination market which is blatantly illegal and contrary to the objectives of the project. These judgements will be made by each member of the NS team individually and projects which are not deemed acceptable by at least 3 of the 5 team members will not be funded, even if that means burning coins. Some questions which are useful in the evaluation of acceptability:
 
 * Does the project seem more likely to benefit the PKT ecosystem than to harm it ?
 * Does the project seem unlikely to cause significant damage to the perceived legitimacy of the Network Steward ?
 * Does the project seem likely to be a better use of resources than burning the coins ? (note that burning coins causes deflation which increases the value of all existing coins)
 
+## Milestone reports
+Milestone reports for ongoing projects must be submitted along with existing projects, the NS will have a
+simple majority vote on whether to accept the milestone and if it is accepted then it is paid out from
```

Comment:
If this is not already defined somewhere then a process should exist by which a milestone decision is reached, otherwise this reads very much like it is merely at the whim of the NS.

Reply:
So there are a couple of points here:

1. Delay: The issue of coins we "were going to pay" getting burned doesn't really work, because so far every payment we've made was at the moment when coins were just about to burn or were already burning. The problem I have with delay is that an applicant might propose a project and then wait to see if the price of PKT goes up and then deliver the project later if it does - so what they're getting is a [call option](https://www.investopedia.com/terms/c/call.asp) which is essentially free. There are two ways we could prevent this:
  * Kill projects which are delayed more than twice their projected timeline -- but by this rule the block explorer would have been killed, and I would like to believe it is a poster child of a well executed project. We don't really need projects to be delivered quickly and if we allow people lots of time then they might give us a lot more results in the end.
  * Re-value projects based on the current price of PKT -- but this requires a market to exist and it also raises the question: "what if the price goes down", in which case I think we would want to allow people to apply for an increase in the payout or abandon the project.
2. Unacceptable Milestones: If the applicant submits a milestone report which does not meet the success criteria laid out or otherwise does not represent what a reasonable person would consider to be a correctly delivered project, then we have a right to refuse it and insist on additional work or in a more extreme case, kill the project. There's really no way that this can be not "at the whim of the NS" because trying to codify what is a properly delivered project is opening up a [rules lawyering](https://en.wikipedia.org/wiki/Rules_lawyer) can of worms where unscrupulous applicants will attempt to push right up to the line without crossing it or even seek loopholes.

That said, I'm thinking we should probably accept and pay out project milestones even if there is no ongoing projects because it doesn't seem to take that long to validate a milestone report and for myself, I wouldn't mind getting paid out quickly, I'm sure others will feel the same.